dessia_common/exports.py

```diff
@@ -253,3 +256,100 @@ def autosize_sheet_columns(sheet, min_width=5, max_width=30):
             if width > 0:
                 adjusted_width = min((width + 0.5), max_width)
                 sheet.column_dimensions[column].width = adjusted_width
+
+
+class MarkdownWriter:
+    def __init__(self, print_limit: int = 25, table_limit: int = 12):
+        self.print_limit = print_limit
+        self.table_limit = table_limit
+
+    def _object_titles(self):
+        return ['Attribute', 'Type', 'Value'] #, 'Subvalues']
+
+    def _sequence_to_str(self, value: List[Union[list, dict, set]]):
+        in_values = value
+        if len(value) == 0:
+            return f"empty {type(value).__name__}"
+
+        printed_string = f"{len(value)} "
+        if isinstance(value, dict):
+            in_values = list(value.values())
```

Comment:
No it is not really a problem because it is just a print.
The purpose here is to get the string of all classes in printed_string (e.g. : `printed_string = 'class1, class2, class3') and to print the first self.print_limit char of this string in the table